cmd/vela-worker/operate.go

```diff
@@ -23,6 +26,35 @@ func (w *Worker) operate() error {
 	if err != nil {
 		return err
 	}
+	// Define the database representation of the worker
+	// and register itself in the database
+	registryWorker := new(library.Worker)
+	registryWorker.SetHostname(w.Config.Hostname)
+	registryWorker.SetAddress(w.Config.Server.Address)
+	registryWorker.SetRoutes(w.Config.Queue.Routes)
+	registryWorker.SetActive(true)
+	registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
+	w.register(registryWorker)
+	if err != nil {
+		logrus.Error("unable to register worker with the server")
+	}
+
+	// spawn goroutine for phoning home
+	go func() {
+		for {
+			// sleep for the configured time
+			time.Sleep(w.Config.CheckIn)
+
+			// set checking time to now and call the server
+			registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
+			_, _, err := w.VelaClient.Worker.Update(registryWorker.GetHostname(), registryWorker)
+
+			// if unable to update the worker, log the error but allow the worker to continue running
+			if err != nil {
+				logrus.Error("unable to update worker in database: %w", err)
```

Comment:
I think we could provide more information in this error message 👍 

Also, we'll want to remove the `%w` as that is a formatting directive only supported in the `fmt` pkg.

To be consistent, it should probably look identical to the [other error message that was written](https://github.com/go-vela/worker/blob/9438a4ab47b5a43d1a08ec0295f40e39b8286ec9/cmd/vela-worker/register.go#L42).

```suggestion
				logrus.Errorf("unable to update worker %s on the server: %v", registryWorker.GetHostname(), err)
```